### PR TITLE
Devrel OKRs

### PR DESCRIPTION
**Pull request by campoy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,3 +130,53 @@
...
+- [ ] Increase followers on Twitter / Facebook /  Linkedin ...
+- [ ] Provide an alternate method to Slack for communication: forum?
+
+### Create a weekly newsletter for announcements, events, curated content, etc
+- [ ] Create the newsletter
+- [ ] Curate content and send newsletter weekly
+- [ ] Reach 100 subscribers
+
+### Increase social engagement with #MLonCode #49
+- [ ] Increase hashtag usage on social media channels by X% (TBD with #48)
```
Comment (eiso) [inline README.md]: @vcoisne can you clarify the meaning of `#48` here

Comment (campoy) [reply README.md]: He meant #48, specifically point one: `[P0] Describe the metrics of devrel success`

File: README.md [hunk 2]
```diff
@@ -117,3 +130,53 @@
...
+- [ ] Define "MLonCode friends" program goals & value proposition
+- [ ] Identify potential members and invite them officially, get 20 to join
+- [ ] Welcome each member personally and provide some swag
+- [ ] Classify members by profile (speakers, researchers, industry/academia, event organizers, etc)
+- [ ] Engage with members and enable them to grow their influence and our message
+
+### Make source{d} a friend of MLonCode academia  
+- [ ] Create a list of all academic events relevant to MLonCode until end of 2019
+  - [ ] Decide level of involvement (attending, presenting, sponsoring, etc)
+- [ ] Release CodeNet with one challenge on top of PGA
```
Comment (bzz) [inline README.md:151]: Sorry if that is a stupid question, but what is CodeNet?

File: README.md [hunk 3]
```diff
@@ -117,3 +130,53 @@
...
+- [ ] Consider and plan/reject creation of competition on Kaggle
+- [ ] 3 citations of source{d} datasets / tech in academia endorsed documents
+- [ ] Develop and organize at least 3 Enterprise workshops in Q3
+
+### Promote current source{d} stack
+- [ ] Write and promote 1 blog post for each project in the stack and how it came to be
+- [ ] Gather list of users of each project and classify them according to their profiles
+- [ ] source{d} Engine and/or Automated Code Review Bot featured at minimum 5 conferences
```
Comment (bzz) [inline README.md]: A bit confused here, what is `Automated Code Review Bot` in `current source{d} stack`? 
Is it a `lookout`?

File: README.md (hunk 2 above)
Comment (vmarkovtsev) [reply README.md:151]: I am +1 this :smile:

File: README.md [hunk 4]
```diff
@@ -117,3 +130,53 @@
...
+- [ ] Identify potential members and invite them officially, get 20 to join
+- [ ] Welcome each member personally and provide some swag
+- [ ] Classify members by profile (speakers, researchers, industry/academia, event organizers, etc)
+- [ ] Engage with members and enable them to grow their influence and our message
+
+### Make source{d} a friend of MLonCode academia  
+- [ ] Create a list of all academic events relevant to MLonCode until end of 2019
+  - [ ] Decide level of involvement (attending, presenting, sponsoring, etc)
+- [ ] Release CodeNet with one challenge on top of PGA
+- [ ] Consider and plan/reject creation of competition on Kaggle
```
Comment (vmarkovtsev) [inline README.md:152]: Last time we asked it was $100,000

File: README.md [hunk 5]
```diff
@@ -117,3 +130,53 @@
...
+- [ ] 3 citations of source{d} datasets / tech in academia endorsed documents
+- [ ] Develop and organize at least 3 Enterprise workshops in Q3
```
Comment (vmarkovtsev) [inline README.md:154]: Does this include TF+Go workshop?

File: README.md (hunk 3 above)
Comment (vmarkovtsev) [reply README.md]: @bzz It is - there was a discussion somewhere on Slack about the name

**Review by vmarkovtsev — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (campoy) [reply README.md:154]: No, this is about workshops of our tech stack.

File: README.md (hunk 2 above)
Comment (campoy) [reply README.md:151]: CodeNet is just an initial name for the MLonCode equivalent of [ImageNet](http://www.image-net.org/).
A huge dataset with challenges that academia can use as a benchmark for their techniques.

File: README.md (hunk 3 above)
Comment (campoy) [reply README.md]: Yeah, we have not decided on the name yet. So lookout, or automated code review, or even assisted code review ... whatever we call it, needs to be promoted 😄

File: README.md (hunk 4 above)
Comment (campoy) [reply README.md:152]: 💰💰💰💰I guess the `reject` part of the OKR is there for something

File: README.md [hunk 6]
```diff
@@ -95,6 +99,15 @@
 - [ ] Service deployed on a production environment
 - [ ] 2 OSS projects besides source{d} enable lookout
 
+##### Announce Lookout to the world
+- [ ] Prepare talk and announcement at GopherCon
+- [ ] Product webpage with description, videos, and instructions on how to use it
```
Comment (ricardobaeta) [inline README.md:105]: Hi @campoy would you be so kind to briefly explain your expectations on this?

Comment (campoy) [reply README.md:105]: Sure, on the new design there will be a page explaining what lookout is, including a quick screencast of how it works (animation works too), basic installation instructions, etc.

Similar to what Google Cloud does with every product, e.g. cloud.google.com/vision. Notice there's even a little demo widget in there, I'd love to have something like that too.

**Pull request merged**